src/Microsoft.DotNet.Helix/Sdk/tools/azure-pipelines/reporter/test_results_reader/__init__.py

```diff
@@ -88,7 +88,7 @@ def read_results(dir):
     for root, dirs, files in os.walk(dir):
         for file_name in files:
             for f in all_formats:
-                if file_name in f.acceptable_file_names:
+                if file_name.endswith(f.acceptable_file_suffixes):
```

Comment:
Instead of making the function return a tuple, just transform the iterable into a tuple.
```suggestion
                if file_name.endswith(tuple(f.acceptable_file_suffixes)):
```

Reply:
Got it.